packages/headless/src/utils/abort-controller-polyfill.ts

```diff
@@ -0,0 +1,13 @@
+export function createAbortController(): AbortController | null {
+  // For nodejs environments only, we want to load the implementation of AbortController from node-abort-controller package.
+  // For browser environments, we need to make sure that we don't use AbortController as it might not be available (Locker Service in Salesforce)
+  // This is not something that can be polyfilled in a meaningful manner.
+  // This is a low level browser API after all, and only JS code inside a polyfill cannot actually cancel network requests done by the browser.
+
+  if (typeof window === 'undefined') {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const nodeAbort = require('node-abort-controller');
```

Comment:
Unrelated, but we will ultimately want to remove this inline require: Webpack and co really dont like it and warn about it at compile time.

Reply:
Removing it now